internals/webhooks/capp_validation_utils.go

```diff
@@ -16,10 +16,15 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	invalidLogSpecTypeMsg = "Invalid LogSpec Type: %s. Valid types are: %s"
+	missingLogConfigMsg   = "%s log configuration is missing required fields: %s"
```

Comment:
Use the message directly in the webhook instead of as a const. Also use %q instead of %s where it makes sense